godot-macros/src/derive_to_variant.rs

```diff
@@ -0,0 +1,231 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+use crate::util::{decl_get_info, DeclInfo};
+use crate::ParseResult;
+use proc_macro2::TokenStream;
+#[allow(unused_imports)]
+use quote::ToTokens;
+use quote::{format_ident, quote};
+use venial::{Declaration, StructFields};
+
+pub fn transform(decl: Declaration) -> ParseResult<TokenStream> {
+    let mut body = quote! {
+
+        let mut root = godot::builtin::Dictionary::new();
+
+    };
```

Comment:
Please remove the empty lines here. (Also other similar trailing/leading lines in a block scope)